test/dx_e2e.js

```diff
@@ -1,56 +1,115 @@
 
-    const  oneEth = new BN("1000000000000000000")
-    const  oneGwei = new BN("10000000000000000")
-    await depositToken.deposit({value: oneEth})
-    await depositToken.approve(instance.address, oneEth)
+    const instance1 = await DxMgnPool.at(await coordinator.dxMgnPool1.call())
+
+    // do the necessary fundings
+    const oneEth = new BN("1000000000000000000")
+    const oneGwei = new BN("10000000000000000")
+    await depositToken.deposit({ value: oneEth })
+    await depositToken.approve(instance1.address, oneEth)
     await depositToken.approve(dx.address, oneEth)
 
     await secondaryTokenMock.givenAnyReturnBool(true)
 
-    await instance.deposit(10)
-    assert.equal(await instance.numberOfParticipations.call(accounts[0]), 1)
+    await instance1.deposit(10)
+    assert.equal(await instance1.numberOfParticipations.call(accounts[0]), 1)
 
     //frist tokenPair needs to be funded first...
-    await dx.deposit(depositToken.address, oneGwei.toString())
-    await dx.deposit(secondaryTokenMock.address, oneGwei.toString())
+    await dx.deposit(depositToken.address, oneGwei.mul(new BN("5")).toString())
+    await dx.deposit(secondaryTokenMock.address, oneGwei.mul(new BN("5")).toString())
     await dx.addTokenPair(
       depositToken.address,
       secondaryTokenMock.address,
       oneGwei.toString(),
       oneGwei.toString(),
       1,
-      10
+      1
     )
-    //  await waitUntilPriceIsXPercentOfPreviousPrice(depositToken, secondaryTokenMock, 1);
+    // ensure that sells go into auction 2, as auction 1 will have been started
+    await increaseTimeBy(60 * 60 * 6)
+    console.log(await timestamp())
+    await coordinator.participateInAuction()
+
+    assert.equal(await dx.sellerBalances.call(depositToken.address, secondaryTokenMock.address, 2, instance1.address), 10)
+    assert.equal(await dx.sellerBalances.call(secondaryTokenMock.address, depositToken.address, 2, instance1.address), 0)
+
+    //closes auction 1
+    await waitUntilPriceIsXPercentOfPreviousPrice(depositToken, secondaryTokenMock, 1);
+    await dx.postBuyOrder(depositToken.address, secondaryTokenMock.address, 1, oneGwei.toString())
+    await dx.postBuyOrder(secondaryTokenMock.address, depositToken.address, 1, oneGwei.toString())
+
+    assert.equal(await dx.getAuctionIndex(depositToken.address, secondaryTokenMock.address),2)
+
+    //claim funds  and post into next auction auction with more
+    await dx.claimSellerFunds(depositToken.address, secondaryTokenMock.address, accounts[0], 1)
+    await dx.claimSellerFunds(secondaryTokenMock.address, depositToken.address, accounts[0], 1)
+    await dx.claimBuyerFunds(depositToken.address, secondaryTokenMock.address, accounts[0], 1)
+    await dx.claimBuyerFunds(secondaryTokenMock.address, depositToken.address, accounts[0], 1)
+    await dx.postSellOrder(depositToken.address, secondaryTokenMock.address, 0, oneGwei.toString())
```

Comment:
Is this required? Who will do this in our model?

Reply:
Other bots have to post sell orders and make sure that prices do not fall too much. These sell orders clear the auction

And I am just doing the claiming, as I want the tokens back for the next auction